Blockzilla/IntroViewController.swift

```diff
@@ -94,7 +94,7 @@ class IntroViewController: UIViewController {
     }
     
     @objc func didTapSkipButton() {
-        Telemetry.default.recordEvent(category: TelemetryEventCategory.firstRun, method: TelemetryEventMethod.skip, object: TelemetryEventObject.onboarding)
+        Telemetry.default.recordEvent(category: TelemetryEventCategory.action, method: TelemetryEventMethod.click, object: TelemetryEventObject.onboarding, value: "skip")
```

Comment:
If this is a new probe, it's fine to change now since we haven't released it yet.